### PR TITLE
Suport as a melodic ROS package

### PR DESCRIPTION
**Pull request by Affonso-Gui**
Adding the commits to support as a melodic package for release.

@k-okada

### CODE REVIEW

**Comment by Affonso-Gui** [comment]
@k-okada

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "slime-repl-ansi-color"]
+	path = slime-repl-ansi-color
+	url = https://github.com/deadtrickster/slime-repl-ansi-color
```
Comment (k-okada) [inline .gitmodules:3]: I think its vetter to specify commit version

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,54 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(euslime)
+
+find_package(catkin REQUIRED)
+find_package(Git QUIET)
+
+catkin_package()
+
+# Update submodules
```
Comment (k-okada) [inline CMakeLists.txt:9]: Do we need this becouse bloom does not download submodule? if so olease add https://github.com/ros-infrastructure/bloom/pull/461 ? https://github.com/ros-infrastructure/bloom/issues/217 ? so if we have newer bloom version , we need to remove this.

File: .gitmodules (hunk 1 above)
Comment (Affonso-Gui) [reply .gitmodules:3]: If my understanding is correct, the version is already (automatically) specified and detached at the latest commit.

https://stackoverflow.com/questions/10914022/how-do-i-check-out-a-specific-version-of-a-submodule-using-git-submodule

File: CMakeLists.txt (hunk 2 above)
Comment (Affonso-Gui) [reply CMakeLists.txt:9]: Couldn't fully understand the discussion on the links, but tried to comment out the update part on the CMakeLists. If this passes the pre-releases builds, then problem solved.

**Pull request merged**